scripts/ci/run_targeting_aura.py

```diff
@@ -120,45 +25,21 @@ def run_notebooks(uri: str, username: str, password: str) -> None:
         raise Exception("Failed to run notebooks")
 
 
-def teardown_instance(access_token: str, db_id: str) -> None:
-    TEARDOWN_MAX_WAIT_TIME = 10
-
-    should_retry = True
-    wait_time = 1
-
-    while should_retry:
-        sleep(wait_time)
-        wait_time *= 2
-
-        response = req.delete(
-            f"https://api-staging.neo4j.io/v1/instances/{db_id}",
-            headers={"Authorization": f"Bearer {access_token}"},
-        )
-
-        if response.status_code == 202:
-            should_retry = False
-
-        should_retry = (response.status_code in [500, 502, 503, 504]) and TEARDOWN_MAX_WAIT_TIME > wait_time
-
-        if should_retry:
-            logging.debug(f"Status code: {response.status_code} - Retrying in {wait_time} s")
-
-    response.raise_for_status()
-
-
 def main() -> None:
-    access_token = get_access_token()
-    logging.info("Access token for creation acquired")
+    client_id = os.environ["AURA_API_CLIENT_ID"]
+    client_secret = os.environ["AURA_API_CLIENT_SECRET"]
+    tenant_id = os.environ.get("TENANT_ID")
+    aura_api = AuraApiCI(client_id=client_id, client_secret=client_secret, tenant_id=tenant_id)
 
     MAX_INT = 1000000
     instance_name = f"ci-build-{sys.argv[2]}" if len(sys.argv) > 1 else "ci-instance-" + str(rd.randint(0, MAX_INT))
 
-    create_result = create_instance(instance_name, access_token)
+    create_result = aura_api.create_ds_instance(instance_name)
```

Comment:
So this file runs tests and notebooks using just an AuraDS instance?
I suggest renaming the file `run_targeting_aurads`

Reply:
I realise this will need another TC change